src/components/CodeSamplesContainer/CodeSampleStyle.tsx

```diff
@@ -0,0 +1,8 @@
+export default function CodeSampleStyle(): Record<string, string> {
+    return {
+        fontSize: "12px",
+        fontWeight: "400",
+        lineHeight: "16px",
+        textAlign: "left"
```

Comment:
We're missing some of the formatting listed in Figma
```suggestion
        fontSize: "12px",
        fontWeight: "400",
        lineHeight: "16px",
        textAlign: "left",
        borderRadius: "8px",
        fontFamily: "Roboto",
        overflowY: "auto",
        padding: "24px"
```